README.md

```diff
@@ -323,8 +323,7 @@ var Axis = React.createClass({
   renderAxis: function() {
     var props = this.props;
     var node  = this.getDOMNode();
-    var scale = props.scale;
-    var axis = d3.svg.axis().orient(props.orient).ticks(5).scale(scale);
+    var axis = d3.svg.axis().orient(props.orient).ticks(5).scale(props.scale);
```

Comment:
Found that scale variable was not necessary and since props.orient did not use a variable, that props.scale should not use a variable either